config/default.yml

```diff
@@ -907,6 +907,12 @@ RSpec/Rails/AvoidSetupHook:
   VersionAdded: '2.4'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/AvoidSetupHook
 
+RSpec/Rails/HaveHttpStatus:
+  Description: Checks that tests use `have_http_status` instead of equality matchers.
+  Enabled: pending
+  VersionAdded: '2.12'
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/HaveHttpStatus
```

Comment:
I think we need to add a `SafeAutoCorrect: false` here, since we are not 100% sure that the `response` in `response.status` is an HTTP response in a controller test.

Reply:
Fixed in https://github.com/rubocop/rubocop-rspec/pull/1307/commits/9608f3c6fa95b7e987de576886c2181210151a0a